src/components/timer/Timer.js

```diff
@@ -1,4 +1,5 @@
-import React from 'react'
+import React from 'react';
```

Comment:
I am going with the style where I omit semicolons as javascript does not require them